eventually/static/src/mainRouter.js

```diff
@@ -4,12 +4,14 @@ import Home from './containers/Home';
 import Profile from './containers/profile/Profile';
```

Comment:
do we need this component in the containers and router?
maybe it makes sense to move it to components and we would use that component in varies pages.

Reply:
done